### PR TITLE
Add PR and Issue templates

### LINKED ISSUE #160: Use Issue and Pull Request templates
A good way to ensure that issues we receive will be understandable. Also add a contributon.md

### PR DESCRIPTION
**Pull request by benbieler**
### What's new?
 - added Issue/PR templates

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Contribution guidelines:
+
+- Be polite.
+- Branch naming convention: `{ IssueID }-short-description`.
+- Commit naming convention: `{ IssueID } { Issue Type (e.g. Enhancement, Bug, etc.) } short description`
+- Run `npm test`
+- Run `xo --fix` if you have any [coding style](https://github.com/sindresorhus/xo) issues. They will be automatically fixed for you.
```
Comment (sindresorhus) [inline .github/CONTRIBUTING.md]: Many issues will be fixed, but not all.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+##### When opening a pull request:
+1. Make sure you squashed all your committed into one. A clean commit history is important.
```
Comment (bobthecow) [inline .github/CONTRIBUTING.md]: Now that [GitHub supports commit squashing on merge](https://github.com/blog/2141-squash-your-commits), we can enforce this at the repo level and drop it from this entirely?

Comment (Ma27) [inline .github/CONTRIBUTING.md]: what if someone releases a bigger refactoring? When having bigger changes it's usually better to have logical chunks, so enforcing squashes for any PR is IMHO not the best solution

**Comment by benbieler** [comment]
Well, this project should be kept as simple as posible, so a BIG refactoring is not in sight.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (benbieler) [inline .github/CONTRIBUTING.md]: Well, this project should be kept as simple as posible, so a BIG refactoring is not in sight.

**Pull request merged**